app/controllers/media_objects_controller.rb

```diff
@@ -336,7 +336,7 @@ def show
       format.json do
         response_json = @media_object.as_json
         response_json.except!(:files, :visibility, :read_groups) unless current_ability.is_administrator?
-        render json: response_json.to_json
+        render json: response_json.to_json(:except => :captions)
```

Comment:
Some caption files not UTF-8 therefore causes failures only in the JSON output format (i.e., works with media player). This API is not used. Content clean-up will happen after export. Captions can be exported in a different manner.

Reply:
The `unless` slightly changes the scope, would prefer have the two exceptions sets separate with the `captions` covering the entire scope. Feels wrong to add tech debt but the real solution is to fix the need behind including captions here (white screen of death when caption files include non utf-8 characters).
```
#line 338
response_json.except!(:files, :visibility, :read_groups) unless current_ability.is_administrator?
```